lib/ansible/modules/network/cli/cli_config.py

```diff
@@ -184,7 +183,7 @@
 
 RETURN = """
 commands:
-  description: The set of commands that will be pushed to the remote device
+  description: The set of commands that will be pushed to the remote device. Note: The commands will be returned only for platforms that have C(supports_onbox_diff) set to I(false). For other platforms, using the C(--diff) option with the playbook will return the difference in configuration.                
```

Comment:
The sentence can be rephrased to `The commands will be returned only for platforms that does not support onbox diff. The C(--diff) option with the playbook will return the difference in configuration for devices that has support for onbox diff.`